api/proto/teleport/machineid/v1/bot_instance.proto

```diff
@@ -83,6 +84,14 @@ message BotInstanceStatusHeartbeat {
   string architecture = 8;
   // The OS of the host that `tbot` is running on, determined by runtime.GOOS.
   string os = 9;
+  // Identifies the external updater process.
+  string external_updater = 10;
+  // Identifies the external updated version. Empty if no updater is configured.
+  string external_updater_version = 11;
+  // Information provided by the external updater, including the update group
+  // and updater status.
+  types.UpdaterV2Info updater_info = 12;
```

Comment:
@hugoShaka  I've added these fields (based the inventory hello message) but am not currently populating them.

I took a look at re-using `agent.ReadHelloUpdaterInfo` but I wasn't sure what we should do about the DBPID, etc?